### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by rafaelcestti**
home page section

### CODE REVIEW

File: src/app/home/(UpcomingWorkshop)/index.tsx
Comment (styled-man) [inline src/app/home/(UpcomingWorkshop)/index.tsx:1]: This file is not used, it should be deleted

File: src/app/home/HeroSection.tsx
Comment (styled-man) [inline src/app/home/HeroSection.tsx:1]: This file does not need to be included as it is not being imported anywhere

File: src/app/home/Mission.tsx
Comment (styled-man) [inline src/app/home/Mission.tsx:1]: This file is not being imported, it should be deleted

**Review by styled-man — CHANGES_REQUESTED** [review]

File: src/app/home/(LiveEvents)/EventSlideshow.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        <div>
+          <SubTitle className="text-4xl leading-tight">
+            {props.currentEvent.name}
+          </SubTitle>
+          <Text className="mt-2">{props.currentEvent.description}</Text>
+        </div>
+
+        <Link
+          href={"https://google.com"}
```
[the remark this answers is not in the document]
Comment (rafaelcestti) [reply src/app/home/(LiveEvents)/EventSlideshow.tsx:49]: there is no proper link, only link from Notion is the Luma link and thats already in the event card. Should we link to /events route?

File: src/app/home/(LiveEvents)/EventSlideshow.tsx
Comment (styled-man) [inline src/app/home/(LiveEvents)/EventSlideshow.tsx:1]: image

Add an overflow scroll on the y axis for smaller screens

Comment (rafaelcestti) [reply src/app/home/(LiveEvents)/EventSlideshow.tsx:1]: made the height fit on smaller screens

File: src/app/home/(LiveEvents)/EventSlideshow.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (styled-man) [reply src/app/home/(LiveEvents)/EventSlideshow.tsx:46]: I'm gonna make this just redirect to the same rsvp link for the event

**Review by styled-man — APPROVED** [review]

**Review by 02danieljonas — APPROVED** [review]
Website works for me!!!!

**Pull request merged**